Doc/using/cmdline.rst

```diff
@@ -112,6 +112,79 @@ source.
        python -m timeit -s 'setup here' 'benchmarked code here'
        python -m timeit -h # for details
 
+   Other modules include
```

Comment:
I have to agree with @rhettinger that this is not the right place to document this.

Reply:
Would you say that such a list should be on some other page instead, or that it shouldn't be on the official docs at all?

Reply:
I feel that it doesn't make sense for such a list to exist. What would you do with such a list? It's like making a list of all UNIX commands that have a `-j` flag.

Reply:
> It's like making a list of all UNIX commands that have a -j flag.

Who is a target audience of the tools?

Currently the documentation is dedicated to Python programmers and CPython plugin authors. The most of tools, however, seems to be for more broad audience like sysadmin users. Should we force them to sift through a sizable `man(3)`-like docs while they, expecting just `man(1)`, need nothing more than simple advertisement of available CLI tools to keep them on the belt?

Reply:
That's as good an argument against inluding this list as any, because half the commands supported here are of no use to sysadmin users (unless they are also Python programmers). If you want to post this list (or something like it) in a blog or on realpython.com, feel free, but the Python docs are for Python programmers.